source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -366,6 +370,346 @@ the bulk operation:
+
+To modify the behavior of your bulk write operation, pass a ``ClientBulkWriteOptions``
+instance to the ``BulkWrite()`` method.
+
+The ``ClientBulkWriteOptions`` type allows you to configure options by using the
+following methods:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``SetBypassDocumentValidation()`` 
+     - | Specifies whether the operation can opt-out of document level validation.
+       | Default: ``false``
+
+   * - ``SetOrdered()``
+     - | Specifies whether the driver stops performing write operations after an error occurs. 
+       | Default: ``true``
+
+   * - ``SetComment()``
+     - | Specifies a comment to attach to the operation.
+       | Default: ``nil``
+
+   * - ``SetLet()``
+     - | Specifies parameters for update and delete operations, which improves command
+         readability by separating the variables from the query text.
+       | Default: none
+
+   * - ``SetWriteConcern()``
+     - | Specifies the write concern for the operations. 
+       | Default: ``nil``
+
+   * - ``SetVerboseResults()``
+     - | Specifies whether detailed information about each successful operation is
+         included in the result.
+       | Default: ``false``
+
+Client-Level Return Value
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``BulkWrite()`` method returns a ``ClientBulkWriteResult`` type, which
+includes information about the bulk operation.
+
+The ``ClientBulkWriteResult`` type contains the following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``InsertedCount``
+     - The number of documents inserted.
+
+   * - ``MatchedCount``
+     - The number of documents matched by the :ref:`query filter <golang-query-filter-definition>` in update and replace operations.
+
+   * - ``ModifiedCount``
+     - The number of documents modified by update and replace operations.
+
+   * - ``DeletedCount``
+     - The number of documents deleted.
+   
+   * - ``UpsertedCount`` 
+     - The number of documents :ref:`upserted <golang-upsert-definition>` by update and replace operations.
+
+   * - ``InsertResults`` 
+     - A map of an operation index to the ``_id`` value of each inserted document.
+
+   * - ``UpdateResults`` 
+     - A map of an operation index to the ``_id`` value of each updated document.
+
+   * - ``DeleteResults`` 
+     - A map of an operation index to the ``_id`` value of each deleted document.
+
+   * - ``Acknowledged`` 
+     - A boolean value that indicates whether the write operation was acknowledged.
+
+   * - ``HasVerboseResults`` 
+     - A boolean value that indicates whether the result contains detailed results.
+
+.. _golang-write-model-client:
+
+Define Client Bulk Write Models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To specify the write operations for your bulk operation on multiple
+namespaces, create a ``ClientWriteModel`` for each insert,
+replace, update, or delete.
```

Comment:
S: add a sentence describing how the first two parameters for each model supply the target namespace